Formula/bubblewrap.rb

```diff
@@ -0,0 +1,38 @@
+class Bubblewrap < Formula
+  desc "Unprivileged sandboxing tool for Linux"
+  homepage "https://github.com/containers/bubblewrap"
+  url "https://github.com/containers/bubblewrap/releases/download/v0.4.1/bubblewrap-0.4.1.tar.xz"
+  sha256 "b9c69b9b1c61a608f34325c8e1a495229bacf6e4a07cbb0c80cf7a814d7ccc03"
+  license "LGPL-2.0-or-later"
+
+  head do
+    url "https://github.com/containers/bubblewrap.git", branch: "master"
+    depends_on "autoconf" => :build
+    depends_on "automake" => :build
+  end
+
+  depends_on "docbook-xsl" => :build
+  depends_on "libxslt"     => :build
+  depends_on "strace" => :test
+  depends_on "libcap"
+  depends_on :linux
+
+  def install
+    system "autoreconf", "-fvi" if build.head?
+    system "./configure", *std_configure_args, "--disable-silent-rules",
+           "--with-bash-completion-dir=#{bash_completion}"
+
+    # Use docbook-xsl's docbook style for generating the man pages:
+    inreplace "Makefile" do |s|
+      s.gsub! "http://docbook.sourceforge.net/release/xsl/current",
+              "#{Formula["docbook-xsl"].opt_prefix}/docbook-xsl"
+    end
+
+    system "make", "install"
+  end
+
+  test do
+    system "#{bin}/bwrap", "--version"
+    assert_match "clone", shell_output("strace -e clone #{bin}/bwrap --bind / / /bin/echo hi 2>&1||:")
```

Comment:
I think this is to allow non-zero returns. But if that is the case you should specify the optional second returncode argument to `shell_output`. 
```suggestion
    assert_match "clone", shell_output("strace -e clone #{bin}/bwrap --bind / / /bin/echo hi 2>&1")
```

Reply:
Sean, I have to explain to you what bubblewrap does.

Please listen to me:

bubbllewrap makes this systemcall, strace shows this:

clone(child_stack=NULL, flags=CLONE_NEWNS|CLONE_NEWUSER|SIGCHLD)

But, depending on if the docker container, or CI environment, or the user's host (whatever runs "brew test") is running a kernel which supports the CLONE_NEWNS|CLONE_NEWUSER flags for normal users, this call will succeed or not.

The test cannot know if the kernel will answer this call with -EPERM or a success, which determines the exit code of bwrap in any test except a version string or help test output test.

Hence, there are tree possiblities:
1. Check that bwrap does what it is supposed to do until the point where it no longer is in control (which is what this test does)
2. Check what the kernel will answer to such system call and expect accordingly. For this, we have to test the kernel with the tool `unshare` before, giving us the expected return code)
3. Just check the version.

Are you OK with 1, 2, or 3?

Reply:
I found a way to use the optional second returncode argument to shell_output as requested. I hope this is is finally!